genshin/models/genshin/diary.py

```diff
@@ -154,9 +162,14 @@ class StarRailDiaryAction(APIModel):
 
     action: str
     action_name: str
-    time: datetime.datetime = Aliased(timezone=8)
+    time: datetime.datetime
     amount: int = Aliased("num")
 
+    @pydantic.field_validator("time")
+    @classmethod
+    def __add_timezone(cls, value: datetime.datetime) -> datetime.datetime:
+        return value.replace(tzinfo=CN_TIMEZONE)
```

Comment:
I believe a custom `Timezone` type with `pydantic.PlainValidator` could be used.